app/src/renderer/vuex/modules/delegation.js

```diff
@@ -58,6 +58,7 @@ export default ({ node }) => {
     },
     resetSessionData({ rootState }) {
       rootState.delegation = JSON.parse(JSON.stringify(emptyState))
+      console.log("rest", rootState.delegation)
```

Comment:
Was this intended for temporary debugging?

Reply:
Removed it